Squablr/View Controllers/ChatViewController.m

```diff
@@ -0,0 +1,31 @@
+//
+//  ChatViewController.m
+//  Squablr
+//
+//  Created by Zeke Reyes on 7/8/22.
+//
+
+#import "ChatViewController.h"
+
+@interface ChatViewController ()
+
+@end
+
+@implementation ChatViewController
```

Comment:
I thought we're not having a chat view anymore? Just a list of profiles you matched with correct? So this should be something like MatchesViewController.